src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+### Off-chain attestations with persistent access {#offchain-attestations-with-persistent-access}
+
+Under this arrangement attestations are transformed into JSON files and stored off-chain (ideally on a [decentralized cloud storage](/developers/docs/storage/) platform, such as IPFS or Swarm). However, a hash of the JSON file is stored on-chain and linked to a DID via an on-chain registry. The associated DID could either be that of the issuer of the attestation or the recipient. 
+
+This approach enables attestations to gain blockchain-based persistence, while keeping claims information encrypted and verifiable. It also allows for selective disclosure since the holder of the private key can decrypt the information. 
+
+### On-chain attestations {#onchain-attestations}
+
+On-chain attestations are held in [smart contracts](/developers/docs/smart-contracts/) on the Ethereum blockchain. The smart contract (acting as a registry in this case) will map an attestation to a corresponding on-chain DID (i.e., a public key). 
+
+Here's an example to show on-chain attestations might work in practice:
+
+1. A company (XYZ Corp.) plans to sell ownership shares using a smart contract, but only wants buyers that have completed a background check.
+
+2. XYZ Corp. can have the company handling the background checks issue on-chain attestations on Ethereum. This attestation certifies that an individual (represented by a DID) has completed the check, but doesn't expose any personal information. 
+
+3. The smart contract handling the sales of shares can call the registry contract for identities of screened buyers. This makes it possible for the smart contract to determine who is permitted to buy shares or not. 
+
+## Benefits of decentralized identity {#benefits-of-decentralized-identity} 
+
+1. Decentralized identity increases increases individual control of identifying information. Decentralized identifiers and attestations can be verified without relying on centralized authorities and third-party services. 
+
+2. Decentralized identity solutions facilitates a trustless, seamless, and privacy-protecting method for verifying and managing user identity. 
+
+3. Decentralized identity harnesses blockchain technology, which creates trust between different parties and provides cryptographic guarantees to prove the validity of attestations.
+
+4. Decentralized identity makes identity data portable. Users store attestations and identifiers in mobile wallet and can share with any party of their choice. Decentralized identifiers and attestations are not locked into the database of the issuing organization. 
+
+## Decentralized identity use-cases {#decentralized-identity-use-cases}
+
+Decentralized identity has many potential use-cases:
+
+### 1. Universal dApp logins {#universal-dapp-logins} 
```

Comment:
We prefer the 'dapp' stylisation on ethereum.org

```suggestion
### 1. Universal dapp logins {#universal-dapp-logins} 
```

Wondering if we should drop dapp entirely here. Doesn't need to be a dapp to benefit from something like SIWE

```suggestion
### 1. Universal logins {#universal-dapp-logins} 
```

Reply:
That makes sense (Web2 companies can also use decentralized identity, too). I'll replace "dapps" with "service providers" if that works.